app/assets/javascripts/index_map.js

```diff
@@ -45,15 +44,29 @@ function initMap() {
 
   var mc = new MarkerClusterer(map, unit_markers);
 
-  initHeatMap(map, heatMapData);
-
+  initHeatMap(map);
 }
 
-function initHeatMap(map, heatMapData) {
+function initHeatMap(map) {
+  var heatMapPoint;
+  var heatMapData = [];
+  var unit = [];
+  var monthly_rent;
```

Comment:
Identifier 'monthly_rent' is not in camel case.<br>'monthly_rent' is defined but never used.